package.json

```diff
@@ -1,25 +1,21 @@
 {
   "name": "coala-bears",
-  "version":"0.8.0",
+  "version": "0.10.0.dev99999999999999",
   "dependencies": {
-    "alex": "~3",
-    "autoprefixer": "~6",
-    "babel-eslint": "~6",
-    "bootlint": "~0",
-    "coffeelint": "~1",
-    "complexity-report": "~2.0.0-alpha",
-    "csslint": "~1",
-    "dockerfile_lint": "~0",
-    "eslint": "~2",
-    "eslint-plugin-import": "~1",
-    "happiness":"~7.1.2",
-    "jshint": "~2",
-    "postcss-cli": "~2",
-    "remark-cli": "~2",
-    "remark-lint": ">=5.1.0",
-    "tslint": "~3",
-    "typescript": ">=1.7.3",
-    "ramllint": "~1.2.2",
-    "write-good": "~0.9.1"
-  }
+  "alex": "~3",
```

Comment:
Before the earlier review the indent was `4` , so I changed to `2`. I think 4 matches with the existing `package.json`. Also Couldn't get indent the 2nd last closing bracket `}` as it's auto-generated by `{{dependencies | jsonify(indent=4, sort_keys=True)}}`